rest/database_init_manager.go

```diff
@@ -72,9 +73,9 @@ func (m *DatabaseInitManager) InitializeDatabase(ctx context.Context, startupCon
 		delete(m.workers, dbConfig.Name)
 	}
 
-	base.InfofCtx(ctx, base.KeyAll, "Starting new async initialization for database %s ...",
+	base.InfofCtx(ctx, base.KeyAll, "Starting new initialization for database %s ...",
 		base.MD(dbConfig.Name))
-	couchbaseCluster, err := CreateBootstrapConnectionFromStartupConfig(ctx, startupConfig, base.PerUseClusterConnections)
+	couchbaseCluster, err := createBootstrapConnectionWithCredentials(ctx, startupConfig, dbConfig.Username, dbConfig.Password, base.PerUseClusterConnections)
```

Comment:
The interesting thing about this circumstance is that the old code didn't support per bucket credentials here:

```
couchbaseCluster, err := CreateBootstrapConnectionFromStartupConfig(ctx, startupConfig, base.PerUseClusterConnections)
```

and it would always use the credentials from the last database defined https://github.com/couchbase/sync_gateway/blob/main/rest/config_legacy.go#L156 for all databases. In practice, this code path only is used with capella where bootstrap connection is defined, so this was never an issue before. I think to be technically correct this needs to account for possibly a different server as well.

Reply:
This should be fixed now